test/Committee.ts

```diff
@@ -148,6 +155,52 @@ describe("Committee", () => {
             .should.be.revertedWithCustomError(committee, "AccessManagedUnauthorized");
     });
 
+    it("should set rng contract", async () => {
+        const {committee} = await whitelistedAndStakedAndHealthyNodes();
+        const rng = await ethers.deployContract("MockRNG");
+        await rng.waitForDeployment();
+        const [, hacker] = await ethers.getSigners();
+
+        await committee.setRNG(rng);
+
+        // This should revert if address is invalid
+        await committee.setRNG(hacker).should.be.revertedWithCustomError(committee, "InvalidContract");
+
+        expect(await committee.skaleRng()).to.equal(await rng.getAddress());
+    });
+
+   it("should select committee with custom rng", async function () {
+        this.timeout(600000); // 10 minutes timeout. DKG requires a lot of time
```

Comment:
No need to run DKG.
The call happens only in the `select` function. Let's save some time.